### PR TITLE
New game! 4 in a Row

### PR DESCRIPTION
**Pull request by fravic**
This adds 4 in a Row (aka Connect Four) -- A quick, easy board game in which two players compete to get four pieces in a line. The implementation supports rocketcrab's query params for player name, so users won't have to input their name twice.

Source code for 4 in a Row available [here](https://github.com/fravic/board-game-server)

Screen Shot 2021-01-02 at 5 05 49 PM

Screen Shot 2021-01-02 at 4 58 47 PM

### CODE REVIEW

**Comment by jonmumm** [comment]
+1! Would really love to play this game on rocketcrab!

**Review by jonmumm — APPROVED** [review]

**Comment by tannerkrewson** [comment]
Wow nice, this looks great! I'll need to finish #85 first

**Comment by robbiet480** [comment]
This is awesome! Looking forward to #85 and this getting merged!

File: config/games/4inarow.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { ServerGame } from "../../types/types";
+import { postJson } from "../../utils/utils";
+
+const game: ServerGame = {
+    id: "4inarow",
+    name: "4 in a Row",
+    author: "fravic",
+    basedOn: {
+        game: "Connect Four",
```
Comment (mavit) [inline config/games/4inarow.ts:9]: We can supply more “based on” information:

```suggestion
        game: "Connect Four",
        link: "https://en.wikipedia.org/wiki/Connect_Four",
        bggId: 2719,
```

Comment (fravic) [reply config/games/4inarow.ts:9]: Thanks, updated!

**Pull request closed without merging**